autogpt_platform/backend/backend/server/routers/v1.py

```diff
@@ -153,10 +153,10 @@ async def update_user_email_route(
     "/auth/user/preferences",
     summary="Get notification preferences",
     tags=["auth"],
-    dependencies=[Depends(auth_middleware)],
+    dependencies=[Security(requires_user)],
```

Comment:
make sure this is secure if only one of these two are put on 

ex
only get_user_id